libs/feature/search/src/lib/state/effects.spec.ts

```diff
@@ -49,22 +49,81 @@ import { FavoritesService } from '@geonetwork-ui/api/repository'
 import { PlatformServiceInterface } from '@geonetwork-ui/common/domain/platform.service.interface'
 
 const defaultSearchState = initialState[DEFAULT_SEARCH_KEY]
-const stateWithSearches = {
+const stateWithSearches: SearchState = {
   ...initialState,
   [DEFAULT_SEARCH_KEY]: {
     ...defaultSearchState,
     config: {
       ...defaultSearchState.config,
       aggregations: SAMPLE_AGGREGATIONS_PARAMS(),
+      filters: {
+        resourceType: {
+          service: false,
+          map: false,
+          'map/static': false,
+          mapDigital: false,
+        },
+        format: {
+          PDF: true,
+        },
+      },
+    },
+    params: {
+      ...defaultSearchState.params,
+      filters: {
+        resourceType: {
+          dataset: true,
+        },
+        format: {},
+        publicationYearForResource: {
+          '2024': true,
+        },
+      },
     },
   },
-  main: {
```

Comment:
Your testes fail because you remove the search state for the `main` search.
Those tests specifically test that the action are forwarded with the input main id (eg. `main`)